### PR TITLE
Added Brunch build tool to the docs

### PR DESCRIPTION
**Pull request by beznosd**
Added Brunch to this articles:
https://facebook.github.io/react/docs/installation.html#development-and-production-versions 
https://facebook.github.io/react/docs/optimizing-performance.html#use-the-production-build

### CODE REVIEW

File: docs/docs/installation.md [hunk 1]
```diff
@@ -99,6 +99,11 @@ By default, React includes many helpful warnings. These warnings are very useful
 
 If you use [Create React App](https://github.com/facebookincubator/create-react-app), `npm run build` will create an optimized build of your app in the `build` folder.
 
+#### Brunch
+
+Just add flag `-p` to the command to create an optimized production build.
+For example: `brunch build -p`. See more details in [Brunch docs](http://brunch.io/docs/commands) commands section.
```
Comment (gaearon) [inline docs/docs/installation.md]: Does this ensure `process.env.NODE_ENV` get envified?

Comment (beznosd) [inline docs/docs/installation.md]: Definitely yes

Comment (aweary) [inline docs/docs/installation.md]: Let's rephrase this to:

> To create an optimized product build with Brunch, just add the `-p` flag to the `build` command. See the [Brunch docs](http://brunch.io/docs/commands) for more details.

File: docs/docs/optimizing-performance.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ If you're benchmarking or experiencing performance problems in your React apps,
 
 * For Create React App, you need to run `npm run build` and follow the instructions.
 * For single-file builds, we offer production-ready `.min.js` versions.
+* For Brunch, you need to add flag `-p` to the command.
```
Comment (aweary) [inline docs/docs/optimizing-performance.md]: Lets rephrase to:

> For Brunch, you need to add the `-p` flag to the `build` command.

**Comment by aweary** [comment]
Thanks @beznosd, looks great! I realized I had a typo in my recommended changes so I went ahead and fixed it for you. Sorry about that 😄

**Comment by beznosd** [comment]
Cool! It's okay. Thanks @aweary!

**Comment by gaearon** [comment]
@aweary Can you send a PR that reorders them by alphabet, except for CRA which should go first? Otherwise it's a bit odd that Brunch goes first considering it's much less popular in the React community.
